tester/runner.go

```diff
@@ -139,6 +140,30 @@ func (r *Runner) SetCompiler(compiler *ast.Compiler) *Runner {
 	return r
 }
 
+type Builtin struct {
+	Decl *ast.Builtin
+	Func func(*rego.Rego)
+}
+
+// Warning: If you set a new compiler after this method, it overrides the compiler with the registered custom builtins
+func (r *Runner) SetBuiltins(builtinsList []*Builtin) *Runner {
```

Comment:
Considering that this adds new bultins on top of the "base builtins", perhaps `AddBuiltins` would be a better name?